indico/modules/events/registration/controllers/management/reglists.py

```diff
@@ -229,7 +229,8 @@ def _send_emails(self, form):
                                    attachments=attachments)
             signals.core.before_notification_send.send('registration-custom-email', email=email,
                                                        registration=registration, form=form)
-            send_email(email, self.event, 'Registration', log_metadata={'registration_id': registration.id})
+            send_email(email, self.event, 'Registration', user=session.user,
```

Comment:
```suggestion
            send_email(email, self.event, 'Registration', session.user,
```

we pass it as a posarg everywhere else

Reply:
done